extended/extended.c:13:9: error: initialization of 'int' from 'void *' makes integer from point
er without a cast [-Werror=int-conversion]
   13 |         NULL,
      |         ^~~~
extended/extended.c:13:9: note: (near initialization for 'module_methods[0].ml_flags')
cc1: some warnings being treated as errors
error: command '/usr/bin/gcc' failed with exit code 1

diff --git a/tests/masonry/builders/fixtures/extended/extended/extended.c b/tests/masonry/builders/fixtures/extended/extended/extended.c
index 25a028e..8980ea3 100644
--- a/tests/masonry/builders/fixtures/extended/extended/extended.c
+++ b/tests/masonry/builders/fixtures/extended/extended/extended.c
@@ -10,7 +10,7 @@ static PyMethodDef module_methods[] = {
     {
         "hello",
         (PyCFunction) hello,
-        NULL,
+        0,
         PyDoc_STR("Say hello.")
     },
     {NULL}
